cdap-kubernetes/src/main/java/co/cask/cdap/master/environment/k8s/PodInfo.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.master.environment.k8s;
+
+import io.kubernetes.client.models.V1OwnerReference;
+
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+/**
+ * Pod information
+ */
+public class PodInfo {
+  private final String dir;
+  private final String labelsFile;
+  private final String nameFile;
+  private final String namespace;
+  private final String name;
+  private final Map<String, String> labels;
+  private final List<V1OwnerReference> ownerReferences;
+
+  public PodInfo(String dir, String labelsFile, String nameFile, String namespace, @Nullable String name,
+                 Map<String, String> labels, List<V1OwnerReference> ownerReferences) {
+    this.dir = dir;
+    this.labelsFile = labelsFile;
+    this.nameFile = nameFile;
+    this.namespace = namespace;
+    this.name = name;
+    this.labels = labels;
+    this.ownerReferences = ownerReferences;
+  }
+
+  public String getDir() {
+    return dir;
+  }
+
+  public String getLabelsFile() {
+    return labelsFile;
+  }
+
+  public String getNameFile() {
+    return nameFile;
+  }
+
+  public String getNamespace() {
+    return namespace;
+  }
+
+  @Nullable
+  public String getName() {
```

Comment:
What is this name? The CDAP name in the K8s deployment?

Reply:
yeah, will rename it to cdapInstanceName

Reply:
actually, no this is the pod name that is used to get the owner references